Time taken to run the queries on ./repositories/Django/thread2/19network42_badger/19network42-badger-353029b: 2069.808 seconds.
Time taken to run the queries on ./repositories/Django/thread2/BLSQ_openhexa-app/BLSQ-openhexa-app-61e3cde: 2827.429 seconds.
Time taken to run the queries on ./repositories/Django/thread2/BridgeCityBicycleCoop_workstand/BridgeCityBicycleCoop-workstand-955b4a2: 2265.327 seconds.
Time taken to run the queries on ./repositories/Django/thread2/CPChain_cpchain-website/CPChain-cpchain-website-7a09ce7: 2425.932 seconds.
Analysis failed for: ./repositories/Django/thread2/City-of-Helsinki_kerrokantasi/City-of-Helsinki-kerrokantasi-256134d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/City-of-Helsinki_kerrokantasi/City-of-Helsinki-kerrokantasi-256134d -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/City-of-Helsinki_kerrokantasi/City-of-Helsinki-kerrokantasi-256134d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/City-of-Helsinki_kerrokantasi/City-of-Helsinki-kerrokantasi-256134d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/City-of-Helsinki_kerrokantasi/City-of-Helsinki-kerrokantasi-256134d: 203.433 seconds.
Analysis failed for: ./repositories/Django/thread2/City-of-Helsinki_linkedevents/City-of-Helsinki-linkedevents-50a1e67 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/City-of-Helsinki_linkedevents/City-of-Helsinki-linkedevents-50a1e67 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/City-of-Helsinki_linkedevents/City-of-Helsinki-linkedevents-50a1e67-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/City-of-Helsinki_linkedevents/City-of-Helsinki-linkedevents-50a1e67-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/City-of-Helsinki_linkedevents/City-of-Helsinki-linkedevents-50a1e67: 288.994 seconds.
Time taken to run the queries on ./repositories/Django/thread2/DALME_DALME-Online-Database/DALME-DALME-Online-Database-9f0b918: 2652.766 seconds.
Analysis failed for: ./repositories/Django/thread2/Exodus-Privacy_etip/Exodus-Privacy-etip-5f2dc0c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Exodus-Privacy_etip/Exodus-Privacy-etip-5f2dc0c -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m25s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Exodus-Privacy_etip/Exodus-Privacy-etip-5f2dc0c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Exodus-Privacy_etip/Exodus-Privacy-etip-5f2dc0c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Exodus-Privacy_etip/Exodus-Privacy-etip-5f2dc0c: 223.544 seconds.
Analysis failed for: ./repositories/Django/thread2/FahrplanDatenGarten_FahrplanDatenGarten/FahrplanDatenGarten-FahrplanDatenGarten-179a4c9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/FahrplanDatenGarten_FahrplanDatenGarten/FahrplanDatenGarten-FahrplanDatenGarten-179a4c9 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m49s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/FahrplanDatenGarten_FahrplanDatenGarten/FahrplanDatenGarten-FahrplanDatenGarten-179a4c9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/FahrplanDatenGarten_FahrplanDatenGarten/FahrplanDatenGarten-FahrplanDatenGarten-179a4c9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/FahrplanDatenGarten_FahrplanDatenGarten/FahrplanDatenGarten-FahrplanDatenGarten-179a4c9: 268.566 seconds.
Time taken to run the queries on ./repositories/Django/thread2/INK-USC_LEAN-LIFE/INK-USC-LEAN-LIFE-c1cd7df: 2598.436 seconds.
Time taken to run the queries on ./repositories/Django/thread2/InsaLan_langate2000/InsaLan-langate2000-402d132: 2275.435 seconds.
Analysis failed for: ./repositories/Django/thread2/NAL-i5K_genomics-workspace/NAL-i5K-genomics-workspace-013f44d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/NAL-i5K_genomics-workspace/NAL-i5K-genomics-workspace-013f44d -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/NAL-i5K_genomics-workspace/NAL-i5K-genomics-workspace-013f44d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/NAL-i5K_genomics-workspace/NAL-i5K-genomics-workspace-013f44d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/NAL-i5K_genomics-workspace/NAL-i5K-genomics-workspace-013f44d: 242.295 seconds.
Analysis failed for: ./repositories/Django/thread2/OpenImaging_miqa/OpenImaging-miqa-07c4891 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/OpenImaging_miqa/OpenImaging-miqa-07c4891 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/OpenImaging_miqa/OpenImaging-miqa-07c4891-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/OpenImaging_miqa/OpenImaging-miqa-07c4891-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/OpenImaging_miqa/OpenImaging-miqa-07c4891: 294.404 seconds.
Analysis failed for: ./repositories/Django/thread2/Pandemiia_pandemiia/Pandemiia-pandemiia-aca98a2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Pandemiia_pandemiia/Pandemiia-pandemiia-aca98a2 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 52.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Pandemiia_pandemiia/Pandemiia-pandemiia-aca98a2-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Pandemiia_pandemiia/Pandemiia-pandemiia-aca98a2-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Pandemiia_pandemiia/Pandemiia-pandemiia-aca98a2: 177.165 seconds.
Analysis failed for: ./repositories/Django/thread2/PolyglotDevsLondon_sharespots/PolyglotDevsLondon-sharespots-4320721 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/PolyglotDevsLondon_sharespots/PolyglotDevsLondon-sharespots-4320721 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 57.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/PolyglotDevsLondon_sharespots/PolyglotDevsLondon-sharespots-4320721-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 5.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/PolyglotDevsLondon_sharespots/PolyglotDevsLondon-sharespots-4320721-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/PolyglotDevsLondon_sharespots/PolyglotDevsLondon-sharespots-4320721: 168.058 seconds.
Time taken to run the queries on ./repositories/Django/thread2/PremierLangage_premierlangage/PremierLangage-premierlangage-178d254: 2617.252 seconds.
Analysis failed for: ./repositories/Django/thread2/Privex_cryptotoken-converter/Privex-cryptotoken-converter-49147e3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Privex_cryptotoken-converter/Privex-cryptotoken-converter-49147e3 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Privex_cryptotoken-converter/Privex-cryptotoken-converter-49147e3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Privex_cryptotoken-converter/Privex-cryptotoken-converter-49147e3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Privex_cryptotoken-converter/Privex-cryptotoken-converter-49147e3: 235.593 seconds.
Time taken to run the queries on ./repositories/Django/thread2/Qexo_Qexo/Qexo-Qexo-5dfa710: 2560.78 seconds.
Time taken to run the queries on ./repositories/Django/thread2/QingdaoU_OnlineJudge/QingdaoU-OnlineJudge-fc2dcda: 2398.643 seconds.
Time taken to run the queries on ./repositories/Django/thread2/RUBi-ZA_JMS/RUBi-ZA-JMS-43e3d19: 2392.875 seconds.
Time taken to run the queries on ./repositories/Django/thread2/Saleor-Multi-Vendor_saleor-multi-vendor/Saleor-Multi-Vendor-saleor-multi-vendor-813933e: 3424.364 seconds.
Analysis failed for: ./repositories/Django/thread2/Servatom_Shealth/Servatom-Shealth-e678214 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Servatom_Shealth/Servatom-Shealth-e678214 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Servatom_Shealth/Servatom-Shealth-e678214-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Servatom_Shealth/Servatom-Shealth-e678214-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Servatom_Shealth/Servatom-Shealth-e678214: 212.875 seconds.
Time taken to run the queries on ./repositories/Django/thread2/StamusNetworks_scirius/StamusNetworks-scirius-f7f1638: 1840.011 seconds.
Analysis failed for: ./repositories/Django/thread2/Studio-Yandex-Practicum_balapanlar/Studio-Yandex-Practicum-balapanlar-41d3f9f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Studio-Yandex-Practicum_balapanlar/Studio-Yandex-Practicum-balapanlar-41d3f9f -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Studio-Yandex-Practicum_balapanlar/Studio-Yandex-Practicum-balapanlar-41d3f9f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Studio-Yandex-Practicum_balapanlar/Studio-Yandex-Practicum-balapanlar-41d3f9f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Studio-Yandex-Practicum_balapanlar/Studio-Yandex-Practicum-balapanlar-41d3f9f: 183.426 seconds.
Analysis failed for: ./repositories/Django/thread2/Swiss-Polar-Institute_spi-media-gallery/Swiss-Polar-Institute-spi-media-gallery-5b82f34 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/Swiss-Polar-Institute_spi-media-gallery/Swiss-Polar-Institute-spi-media-gallery-5b82f34 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m41s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Swiss-Polar-Institute_spi-media-gallery/Swiss-Polar-Institute-spi-media-gallery-5b82f34-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/Swiss-Polar-Institute_spi-media-gallery/Swiss-Polar-Institute-spi-media-gallery-5b82f34-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/Swiss-Polar-Institute_spi-media-gallery/Swiss-Polar-Institute-spi-media-gallery-5b82f34: 235.357 seconds.
Time taken to run the queries on ./repositories/Django/thread2/TyVik_geopuzzle/TyVik-geopuzzle-4b2e20e: 2288.487 seconds.
Analysis failed for: ./repositories/Django/thread2/UnitapApp_unitap-backend/UnitapApp-unitap-backend-ec033ee Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/UnitapApp_unitap-backend/UnitapApp-unitap-backend-ec033ee -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m38s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/UnitapApp_unitap-backend/UnitapApp-unitap-backend-ec033ee-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/UnitapApp_unitap-backend/UnitapApp-unitap-backend-ec033ee-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/UnitapApp_unitap-backend/UnitapApp-unitap-backend-ec033ee: 245.789 seconds.
Time taken to run the queries on ./repositories/Django/thread2/Williano_Invoice-Management-System/Williano-Invoice-Management-System-7ad7619: 1991.4 seconds.
Analysis failed for: ./repositories/Django/thread2/YadavGulshan_MedAlthea/YadavGulshan-MedAlthea-301e0e0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/YadavGulshan_MedAlthea/YadavGulshan-MedAlthea-301e0e0 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/YadavGulshan_MedAlthea/YadavGulshan-MedAlthea-301e0e0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/YadavGulshan_MedAlthea/YadavGulshan-MedAlthea-301e0e0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/YadavGulshan_MedAlthea/YadavGulshan-MedAlthea-301e0e0: 212.714 seconds.
Analysis failed for: ./repositories/Django/thread2/acikyazilimagi_ben-iyiyim/acikyazilimagi-ben-iyiyim-94669d5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/acikyazilimagi_ben-iyiyim/acikyazilimagi-ben-iyiyim-94669d5 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 58.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/acikyazilimagi_ben-iyiyim/acikyazilimagi-ben-iyiyim-94669d5-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/acikyazilimagi_ben-iyiyim/acikyazilimagi-ben-iyiyim-94669d5-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/acikyazilimagi_ben-iyiyim/acikyazilimagi-ben-iyiyim-94669d5: 166.851 seconds.
Analysis failed for: ./repositories/Django/thread2/ajenhl_eats/ajenhl-eats-8ac8d3b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/ajenhl_eats/ajenhl-eats-8ac8d3b -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ajenhl_eats/ajenhl-eats-8ac8d3b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ajenhl_eats/ajenhl-eats-8ac8d3b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/ajenhl_eats/ajenhl-eats-8ac8d3b: 201.642 seconds.
Analysis failed for: ./repositories/Django/thread2/alan-turing-institute_AssurancePlatform/alan-turing-institute-AssurancePlatform-93bfb6b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/alan-turing-institute_AssurancePlatform/alan-turing-institute-AssurancePlatform-93bfb6b -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/alan-turing-institute_AssurancePlatform/alan-turing-institute-AssurancePlatform-93bfb6b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/alan-turing-institute_AssurancePlatform/alan-turing-institute-AssurancePlatform-93bfb6b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/alan-turing-institute_AssurancePlatform/alan-turing-institute-AssurancePlatform-93bfb6b: 207.918 seconds.
Time taken to run the queries on ./repositories/Django/thread2/amadeusproject_amadeuslms/amadeusproject-amadeuslms-9c5cae5: 2836.044 seconds.
Time taken to run the queries on ./repositories/Django/thread2/anshumanv_collabman/anshumanv-collabman-f06f659: 2284 seconds.
Time taken to run the queries on ./repositories/Django/thread2/artefactual_archivematica/artefactual-archivematica-fbad601: 2821.011 seconds.
Time taken to run the queries on ./repositories/Django/thread2/astrikov-d_dartcms/astrikov-d-dartcms-41af3ec: 2045.019 seconds.
Time taken to run the queries on ./repositories/Django/thread2/asyml_stave/asyml-stave-9c600bf: 2315.275 seconds.
Analysis failed for: ./repositories/Django/thread2/augeas_TransMediaLint/augeas-TransMediaLint-1d5003c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/augeas_TransMediaLint/augeas-TransMediaLint-1d5003c -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m55s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/augeas_TransMediaLint/augeas-TransMediaLint-1d5003c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/augeas_TransMediaLint/augeas-TransMediaLint-1d5003c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/augeas_TransMediaLint/augeas-TransMediaLint-1d5003c: 290.665 seconds.
Time taken to run the queries on ./repositories/Django/thread2/basxsoftwareassociation_bread/basxsoftwareassociation-bread-7466ff8: 2472.891 seconds.
Analysis failed for: ./repositories/Django/thread2/bcgov_family-law-act-app/bcgov-family-law-act-app-448b227 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/bcgov_family-law-act-app/bcgov-family-law-act-app-448b227 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/bcgov_family-law-act-app/bcgov-family-law-act-app-448b227-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/bcgov_family-law-act-app/bcgov-family-law-act-app-448b227-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/bcgov_family-law-act-app/bcgov-family-law-act-app-448b227: 320.722 seconds.
Time taken to run the queries on ./repositories/Django/thread2/bounswe_bounswe2022group4/bounswe-bounswe2022group4-8a252ac: 2375.268 seconds.
Analysis failed for: ./repositories/Django/thread2/cas-packone_packone/cas-packone-packone-fbf6ff9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/cas-packone_packone/cas-packone-packone-fbf6ff9 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m36s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/cas-packone_packone/cas-packone-packone-fbf6ff9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/cas-packone_packone/cas-packone-packone-fbf6ff9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/cas-packone_packone/cas-packone-packone-fbf6ff9: 247.923 seconds.
Time taken to run the queries on ./repositories/Django/thread2/ccnmtl_mediathread/ccnmtl-mediathread-afe441c: 2531.921 seconds.
Analysis failed for: ./repositories/Django/thread2/chronhq_backend/chronhq-backend-d928f45 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/chronhq_backend/chronhq-backend-d928f45 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m16s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/chronhq_backend/chronhq-backend-d928f45-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/chronhq_backend/chronhq-backend-d928f45-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/chronhq_backend/chronhq-backend-d928f45: 209.246 seconds.
Time taken to run the queries on ./repositories/Django/thread2/comocheng_ChemCheck/comocheng-ChemCheck-d6805ef: 2602.802 seconds.
Time taken to run the queries on ./repositories/Django/thread2/ddos-clearing-house_ddosdb/ddos-clearing-house-ddosdb-1ff25f7: 2258.295 seconds.
Analysis failed for: ./repositories/Django/thread2/developers-cosmos_Mimasa/developers-cosmos-Mimasa-1ede81d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/developers-cosmos_Mimasa/developers-cosmos-Mimasa-1ede81d -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/developers-cosmos_Mimasa/developers-cosmos-Mimasa-1ede81d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/developers-cosmos_Mimasa/developers-cosmos-Mimasa-1ede81d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/developers-cosmos_Mimasa/developers-cosmos-Mimasa-1ede81d: 295.51 seconds.
Analysis failed for: ./repositories/Django/thread2/eduNEXT_eox-core/eduNEXT-eox-core-92bd3d9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/eduNEXT_eox-core/eduNEXT-eox-core-92bd3d9 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/eduNEXT_eox-core/eduNEXT-eox-core-92bd3d9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/eduNEXT_eox-core/eduNEXT-eox-core-92bd3d9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/eduNEXT_eox-core/eduNEXT-eox-core-92bd3d9: 238.735 seconds.
Time taken to run the queries on ./repositories/Django/thread2/edx_edx-exams/edx-edx-exams-5b45de3: 2333.267 seconds.
Analysis failed for: ./repositories/Django/thread2/favours-io_favours/favours-io-favours-6f26a20 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/favours-io_favours/favours-io-favours-6f26a20 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/favours-io_favours/favours-io-favours-6f26a20-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/favours-io_favours/favours-io-favours-6f26a20-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/favours-io_favours/favours-io-favours-6f26a20: 192.978 seconds.
Time taken to run the queries on ./repositories/Django/thread2/grahamgilbert_Crypt-Server/grahamgilbert-Crypt-Server-8562fb5: 2319.449 seconds.
Time taken to run the queries on ./repositories/Django/thread2/grouprise_grouprise/grouprise-grouprise-74a42dd: 2344.207 seconds.
Time taken to run the queries on ./repositories/Django/thread2/guanquann_Stocksera/guanquann-Stocksera-8453d90: 2599.201 seconds.
Time taken to run the queries on ./repositories/Django/thread2/haystack_murmur/haystack-murmur-d10808a: 2325.848 seconds.
Time taken to run the queries on ./repositories/Django/thread2/hegusung_WebHashcat/hegusung-WebHashcat-12b78c3: 2185.272 seconds.
Analysis failed for: ./repositories/Django/thread2/hmisty_coding/hmisty-coding-a6c30dc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/hmisty_coding/hmisty-coding-a6c30dc -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m59s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/hmisty_coding/hmisty-coding-a6c30dc-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 20.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/hmisty_coding/hmisty-coding-a6c30dc-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/hmisty_coding/hmisty-coding-a6c30dc: 298.464 seconds.
Time taken to run the queries on ./repositories/Django/thread2/ietf-tools_datatracker/ietf-tools-datatracker-44a5dcb: 3734.835 seconds.
Analysis failed for: ./repositories/Django/thread2/ireapps_census/ireapps-census-7893a97 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/ireapps_census/ireapps-census-7893a97 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m40s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ireapps_census/ireapps-census-7893a97-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ireapps_census/ireapps-census-7893a97-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/ireapps_census/ireapps-census-7893a97: 238.88 seconds.
Time taken to run the queries on ./repositories/Django/thread2/jkariscodes_personal-website/jkariscodes-personal-website-ce3065f: 2023.963 seconds.
Time taken to run the queries on ./repositories/Django/thread2/kaleidos-ventures_taiga-back/taigaio-taiga-back-11b5b33: 3089.434 seconds.
Time taken to run the queries on ./repositories/Django/thread2/kimlimjustin_google-form-clone/kimlimjustin-google-form-clone-7bbd234: 2031.272 seconds.
Time taken to run the queries on ./repositories/Django/thread2/klee_klee-web/klee-klee-web-a15cfc8: 2265.643 seconds.
Time taken to run the queries on ./repositories/Django/thread2/libreosteo_LibreOsteo/libreosteo-LibreOsteo-6a517cb: 2056.843 seconds.
Analysis failed for: ./repositories/Django/thread2/mangadventure_MangAdventure/mangadventure-MangAdventure-299e4f4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/mangadventure_MangAdventure/mangadventure-MangAdventure-299e4f4 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m11s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/mangadventure_MangAdventure/mangadventure-MangAdventure-299e4f4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/mangadventure_MangAdventure/mangadventure-MangAdventure-299e4f4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/mangadventure_MangAdventure/mangadventure-MangAdventure-299e4f4: 203.891 seconds.
Analysis failed for: ./repositories/Django/thread2/microsoft_OpenEduAnalytics/microsoft-OpenEduAnalytics-e95b682 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/microsoft_OpenEduAnalytics/microsoft-OpenEduAnalytics-e95b682 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/microsoft_OpenEduAnalytics/microsoft-OpenEduAnalytics-e95b682-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/microsoft_OpenEduAnalytics/microsoft-OpenEduAnalytics-e95b682-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/microsoft_OpenEduAnalytics/microsoft-OpenEduAnalytics-e95b682: 210.656 seconds.
Time taken to run the queries on ./repositories/Django/thread2/ministryofjustice_cla_backend/ministryofjustice-cla_backend-5ffaadd: 2474.203 seconds.
Time taken to run the queries on ./repositories/Django/thread2/modelbrouwers_modelbrouwers/modelbrouwers-modelbrouwers-59e008e: 2466.053 seconds.
Time taken to run the queries on ./repositories/Django/thread2/mozilla_network-pulse-api/mozilla-network-pulse-api-193ddfb: 2241.602 seconds.
Time taken to run the queries on ./repositories/Django/thread2/natbat_pillarpointstewards/natbat-pillarpointstewards-4980240: 2058.075 seconds.
Analysis failed for: ./repositories/Django/thread2/nationalarchives_ds-wagtail/nationalarchives-ds-wagtail-f41093d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/nationalarchives_ds-wagtail/nationalarchives-ds-wagtail-f41093d -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m39s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nationalarchives_ds-wagtail/nationalarchives-ds-wagtail-f41093d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nationalarchives_ds-wagtail/nationalarchives-ds-wagtail-f41093d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/nationalarchives_ds-wagtail/nationalarchives-ds-wagtail-f41093d: 261.13 seconds.
Time taken to run the queries on ./repositories/Django/thread2/nelsonwenner_ecommerce/nelsonwenner-ecommerce-566357e: 2092.557 seconds.
Analysis failed for: ./repositories/Django/thread2/nickheyer_DiscoFlix/nickheyer-DiscoFlix-d4a90d6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/nickheyer_DiscoFlix/nickheyer-DiscoFlix-d4a90d6 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nickheyer_DiscoFlix/nickheyer-DiscoFlix-d4a90d6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nickheyer_DiscoFlix/nickheyer-DiscoFlix-d4a90d6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/nickheyer_DiscoFlix/nickheyer-DiscoFlix-d4a90d6: 198.606 seconds.
Analysis failed for: ./repositories/Django/thread2/nutanix_blueprints/nutanix-blueprints-e8ea3db Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/nutanix_blueprints/nutanix-blueprints-e8ea3db -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nutanix_blueprints/nutanix-blueprints-e8ea3db-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/nutanix_blueprints/nutanix-blueprints-e8ea3db-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/nutanix_blueprints/nutanix-blueprints-e8ea3db: 221.818 seconds.
Time taken to run the queries on ./repositories/Django/thread2/okfn_website/okfn-website-8b69fb9: 1984.409 seconds.
Analysis failed for: ./repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc-results/django_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Custom-session-engine/custom_session_engine.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Custom-session-engine/custom_session_engine.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Custom-session-engine/custom_session_engine.ql.
[1/1 eval 27.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc-results/custom_session_engine.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:145
            throw new Error("Using a custom session engine, so the analysis won't be run");
                  ^

Error: Using a custom session engine, so the analysis won't be run
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:145:19)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/onepercentclub_bluebottle/onepercentclub-bluebottle-0cc7cbc: 298.317 seconds.
Analysis failed for: ./repositories/Django/thread2/onespacemedia_cms/onespacemedia-cms-23ab621 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/onespacemedia_cms/onespacemedia-cms-23ab621 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m38s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/onespacemedia_cms/onespacemedia-cms-23ab621-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/onespacemedia_cms/onespacemedia-cms-23ab621-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/onespacemedia_cms/onespacemedia-cms-23ab621: 262.787 seconds.
Time taken to run the queries on ./repositories/Django/thread2/openedx_edx-proctoring/openedx-edx-proctoring-33a0be0: 2365.182 seconds.
Time taken to run the queries on ./repositories/Django/thread2/openfun_richie-site-factory/openfun-richie-site-factory-fb3d956: 1984.1 seconds.
Analysis failed for: ./repositories/Django/thread2/openwisp_ansible-openwisp2/openwisp-ansible-openwisp2-699ac8e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/openwisp_ansible-openwisp2/openwisp-ansible-openwisp2-699ac8e -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/openwisp_ansible-openwisp2/openwisp-ansible-openwisp2-699ac8e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/openwisp_ansible-openwisp2/openwisp-ansible-openwisp2-699ac8e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/openwisp_ansible-openwisp2/openwisp-ansible-openwisp2-699ac8e: 181.746 seconds.
Time taken to run the queries on ./repositories/Django/thread2/phith0n_mooder/phith0n-mooder-611124d: 2000.712 seconds.
Analysis failed for: ./repositories/Django/thread2/prefeiturasp_SME-Livro-Aberto/prefeiturasp-SME-Livro-Aberto-a333aad Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/prefeiturasp_SME-Livro-Aberto/prefeiturasp-SME-Livro-Aberto-a333aad -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/prefeiturasp_SME-Livro-Aberto/prefeiturasp-SME-Livro-Aberto-a333aad-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/prefeiturasp_SME-Livro-Aberto/prefeiturasp-SME-Livro-Aberto-a333aad-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/prefeiturasp_SME-Livro-Aberto/prefeiturasp-SME-Livro-Aberto-a333aad: 235.151 seconds.
Analysis failed for: ./repositories/Django/thread2/pugpe_pugpe/pugpe-pugpe-e567e5f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/pugpe_pugpe/pugpe-pugpe-e567e5f -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/pugpe_pugpe/pugpe-pugpe-e567e5f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/pugpe_pugpe/pugpe-pugpe-e567e5f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/pugpe_pugpe/pugpe-pugpe-e567e5f: 192.16 seconds.
Time taken to run the queries on ./repositories/Django/thread2/rafsaf_Tribal-Wars-Planer/rafsaf-Tribal-Wars-Planer-cec84bf: 2885.133 seconds.
Time taken to run the queries on ./repositories/Django/thread2/recursecenter_blaggregator/recursecenter-blaggregator-81c83fa: 2249.877 seconds.
Analysis failed for: ./repositories/Django/thread2/rhaamo_StockazIO/rhaamo-StockazIO-e673bf0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/rhaamo_StockazIO/rhaamo-StockazIO-e673bf0 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m23s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/rhaamo_StockazIO/rhaamo-StockazIO-e673bf0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/rhaamo_StockazIO/rhaamo-StockazIO-e673bf0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/rhaamo_StockazIO/rhaamo-StockazIO-e673bf0: 234.194 seconds.
Time taken to run the queries on ./repositories/Django/thread2/rubberdok_indok-web/rubberdok-indok-web-16a2f22: 2218.009 seconds.
Analysis failed for: ./repositories/Django/thread2/simplworld_simpl-games-api/simplworld-simpl-games-api-043fca7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/simplworld_simpl-games-api/simplworld-simpl-games-api-043fca7 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m29s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/simplworld_simpl-games-api/simplworld-simpl-games-api-043fca7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/simplworld_simpl-games-api/simplworld-simpl-games-api-043fca7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/simplworld_simpl-games-api/simplworld-simpl-games-api-043fca7: 232.574 seconds.
Analysis failed for: ./repositories/Django/thread2/siyam04_ODLRS_v.2/siyam04-ODLRS_v.2-f95b577 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/siyam04_ODLRS_v.2/siyam04-ODLRS_v.2-f95b577 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 54.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/siyam04_ODLRS_v.2/siyam04-ODLRS_v.2-f95b577-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/siyam04_ODLRS_v.2/siyam04-ODLRS_v.2-f95b577-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/siyam04_ODLRS_v.2/siyam04-ODLRS_v.2-f95b577: 169.69 seconds.
Time taken to run the queries on ./repositories/Django/thread2/spapas_mailer_server/spapas-mailer_server-17bf5c7: 1884.081 seconds.
Time taken to run the queries on ./repositories/Django/thread2/strongswan_strongTNC/strongswan-strongTNC-592a905: 1880.264 seconds.
Analysis failed for: ./repositories/Django/thread2/subject-f_guyamoe/subject-f-guyamoe-a6fe86b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/subject-f_guyamoe/subject-f-guyamoe-a6fe86b -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m52s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/subject-f_guyamoe/subject-f-guyamoe-a6fe86b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/subject-f_guyamoe/subject-f-guyamoe-a6fe86b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/subject-f_guyamoe/subject-f-guyamoe-a6fe86b: 253.915 seconds.
Time taken to run the queries on ./repositories/Django/thread2/summarizepaper_summarizepaper/summarizepaper-summarizepaper-e2842bb: 2162.407 seconds.
Time taken to run the queries on ./repositories/Django/thread2/swirlai_swirl-search/swirlai-swirl-search-51307e8: 1908.879 seconds.
Analysis failed for: ./repositories/Django/thread2/sympy_sympy_gamma/sympy-sympy_gamma-3da0b17 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/sympy_sympy_gamma/sympy-sympy_gamma-3da0b17 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/sympy_sympy_gamma/sympy-sympy_gamma-3da0b17-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/sympy_sympy_gamma/sympy-sympy_gamma-3da0b17-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/sympy_sympy_gamma/sympy-sympy_gamma-3da0b17: 166.16 seconds.
Time taken to run the queries on ./repositories/Django/thread2/tamuhack-org_Ouroboros/tamuhack-org-Ouroboros-147b51d: 1343.99 seconds.
Analysis failed for: ./repositories/Django/thread2/tlambert03_FPbase/tlambert03-FPbase-abe00da Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/tlambert03_FPbase/tlambert03-FPbase-abe00da -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m49s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/tlambert03_FPbase/tlambert03-FPbase-abe00da-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/tlambert03_FPbase/tlambert03-FPbase-abe00da-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/tlambert03_FPbase/tlambert03-FPbase-abe00da: 241.785 seconds.
Analysis failed for: ./repositories/Django/thread2/toladata_TolaActivity/toladata-TolaActivity-364d5c2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/toladata_TolaActivity/toladata-TolaActivity-364d5c2 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 52.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/toladata_TolaActivity/toladata-TolaActivity-364d5c2-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/toladata_TolaActivity/toladata-TolaActivity-364d5c2-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/toladata_TolaActivity/toladata-TolaActivity-364d5c2: 143.662 seconds.
Analysis failed for: ./repositories/Django/thread2/torchbox_wagtail-torchbox/torchbox-wagtail-torchbox-4754f75 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/torchbox_wagtail-torchbox/torchbox-wagtail-torchbox-4754f75 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 49.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/torchbox_wagtail-torchbox/torchbox-wagtail-torchbox-4754f75-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 5.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/torchbox_wagtail-torchbox/torchbox-wagtail-torchbox-4754f75-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/torchbox_wagtail-torchbox/torchbox-wagtail-torchbox-4754f75: 142.719 seconds.
Analysis failed for: ./repositories/Django/thread2/trias-lab_trias-explorer/trias-lab-trias-explorer-8616f70 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/trias-lab_trias-explorer/trias-lab-trias-explorer-8616f70 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 40.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/trias-lab_trias-explorer/trias-lab-trias-explorer-8616f70-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/trias-lab_trias-explorer/trias-lab-trias-explorer-8616f70-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/trias-lab_trias-explorer/trias-lab-trias-explorer-8616f70: 122.675 seconds.
Time taken to run the queries on ./repositories/Django/thread2/trypromptly_LLMStack/trypromptly-LLMStack-bfd3d77: 1376.701 seconds.
Time taken to run the queries on ./repositories/Django/thread2/ubiquitous-computing-lab_Mining-Minds/ubiquitous-computing-lab-Mining-Minds-86ca7db: 997.832 seconds.
Time taken to run the queries on ./repositories/Django/thread2/uno-isqa-8950_uno-cpi/uno-isqa-8950-uno-cpi-852164b: 1059.6 seconds.
Time taken to run the queries on ./repositories/Django/thread2/usdoj-crt_crt-portal/usdoj-crt-crt-portal-be5a76c: 779.332 seconds.
Time taken to run the queries on ./repositories/Django/thread2/webkom_lego/webkom-lego-45de552: 563.225 seconds.
Analysis failed for: ./repositories/Django/thread2/ychalier_orgapy/ychalier-orgapy-09535c2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/ychalier_orgapy/ychalier-orgapy-09535c2 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 14.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ychalier_orgapy/ychalier-orgapy-09535c2-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/ychalier_orgapy/ychalier-orgapy-09535c2-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/ychalier_orgapy/ychalier-orgapy-09535c2: 41.003 seconds.
Analysis failed for: ./repositories/Django/thread2/zappa_Zappa/zappa-Zappa-4a24cb6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread2/zappa_Zappa/zappa-Zappa-4a24cb6 -l=python -t=11 -ct=2
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 19.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/zappa_Zappa/zappa-Zappa-4a24cb6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 2.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread2/zappa_Zappa/zappa-Zappa-4a24cb6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread2/zappa_Zappa/zappa-Zappa-4a24cb6: 51.395 seconds.
Time taken to run the queries on ./repositories/Django/thread2/zuri-training_Favicon_gen-Team-68/zuri-training-Favicon_gen-Team-68-113f6fe: 422.043 seconds.
Time taken to run the queries on ./repositories/Django/thread2/zuri-training_WF-PriceCompare-24/zuri-training-WF-PriceCompare-24-3c9128d: 366.029 seconds.
Time taken to run the queries and generate the statistics: 142413.069 seconds.
